index.html

```diff
@@ -1,17 +1,119 @@
-<!DOCTYPE html> 
-<html>
-    <head>
-        <meta charset="utf-8" />
-        <title>Leiden Software Developers</title>
-    </head>
-    <body>
-        <h1>Leiden Software Developers</h1>
-        <p>Future home of the Leiden Software Developers group.</p>
-        <p>Join us at:</p>
-        <ul>
-            <li><a href="https://www.meetup.com/leidendevs/">Meetup</a></li>
-            <li><a href="https://leidendevs.slack.com/">Slack</a></li>
-            <li><a href="https://old.reddit.com/r/LeidenDevs">Reddit</a></li>
-        </ul>
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Leiden Software Developers</title>
+    <meta name="title" content="Leiden Software Developers">
+    <meta name="description" content="A place for  people who work or play with software to meet &amp; socialize.">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="https://www.leidendevs.nl/">
+    <meta property="og:title" content="Leiden Software Developers">
+    <meta property="og:description" content="A place for  people who work or play with software to meet & socialize.">
```

Comment:
You missed one ampersand here. Other than that, no more comments!

Reply:
Fixed ☑️